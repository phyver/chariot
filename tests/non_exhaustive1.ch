data nat where Zero : nat | Succ : nat -> nat

val f 0 = ???
  | f (n+2) = ???
