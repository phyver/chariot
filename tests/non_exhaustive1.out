*** function f is not complete
